cinema/views.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+
+from django.http import HttpResponse
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def bus_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer, status.HTTP_201_CREATED)
+
+        return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def bus_info(request, pk):
```

Comment:
<img width="150" alt="image" src="https://user-images.githubusercontent.com/65166203/235110166-7370963f-2c19-46a9-83ae-61f08c1b1bb3.png">
The same error with list
Rename it to movie